functions/public/Invoke-WPFMicrowin.ps1

```diff
@@ -117,14 +118,38 @@ public class PowerManagement {
 			if (Test-Path $driverPath)
 			{
 				Write-Host "Adding Windows Drivers image($scratchDir) drivers($driverPath) "
-				Add-WindowsDriver -Path "$scratchDir" -Recurse -Driver "$driverPath"
+				dism /English /image:$scratchDir /add-driver /driver:$driverPath /recurse | Out-Host
 			}
 			else 
 			{
 				Write-Host "Path to drivers is invalid continuing without driver injection"
 			}
 		}
 
+		if ($importDrivers)
+		{
+			Write-Host "Exporting drivers from active installation..."
+			if (Test-Path "$env:TEMP\DRV_EXPORT")
+			{
+				Remove-Item "$env:TEMP\DRV_EXPORT" -Recurse -Force
+			}
+			New-Item -Path "$env:TEMP\DRV_EXPORT" -ItemType Directory -Force
+			dism /english /online /export-driver /destination="$env:TEMP\DRV_EXPORT" | Out-Host
+			if ($?)
+			{
+				Write-Host "Adding exported drivers..."
+				dism /english /image="$scratchDir" /add-driver /driver="$env:TEMP\DRV_EXPORT" /recurse | Out-Host
+			}
+			else
+			{
+				Write-Host "Failed to export drivers. Continuing without importing them..."
+			}
+			if (Test-Path "$env:TEMP\DRV_EXPORT")
+			{
+				Remove-Item "$env:TEMP\DRV_EXPORT" -Recurse -Force
```

Comment:
I think we should have an option to just export drivers for later use, I've had that on my plate for the longest time.  Maybe you could add a path to directory to export to and not delete it. This is important when you do prepping on a different machine, and maybe you want to trim some drivers and leave only the very necessary drivers.

Reply:
This is already implemented by the change mentioned in your last review